spec/lib/task_helpers/exports/custom_buttons_spec.rb

```diff
@@ -70,7 +70,7 @@
       TaskHelpers::Exports::CustomButtons.new.export(:directory => export_dir)
       file_contents = File.read("#{export_dir}/CustomButtons.yaml")
 
-      expect(YAML.safe_load(file_contents, [Symbol])).to contain_exactly(*custom_button_export_test)
+      expect(YAML.safe_load(file_contents, permitted_classes: [Symbol])).to contain_exactly(*custom_button_export_test)
```

Comment:
I think there's a way to set permitted_classes globally, so we can add most of the reasonable defaults of Date, Time, DateTime, Symbol, etc.

Reply:
This was a fast pass through to see what's busted as it's far easier to comment on code 😉